package.json

```diff
@@ -51,7 +51,7 @@
     "website"
   ],
   "dependencies": {
-    "gatsby-cli": "^2.5.9"
+    "gatsby-cli": "^2.5.12"
```

Comment:
Oh wait, this should go to the website package.

Reply:
41f9b1c